### ISSUE TITLE
New playbook action to increase node group size

### ISSUE THREAD

**Issue by aantn** [robusta-dev/robusta#223]
We can start by supporting only one cloud provider (e.g. EKS) - that's OK

**Comment by aantn** [comment]
Yep, exactly! Which cloud provider were you thinking of working on?

**Comment by wahajXgrid** [comment]
> Yep, exactly! Which cloud provider were you thinking of working on?

Google Cloud
